shimmer/develop/sidebars.ts

```diff
@@ -1,200 +1,216 @@
-        },
-        {
-          label: 'Wallet.rs',
-          type: 'link',
-          href: '/wallet.rs/welcome',
-        },
-        {
-          label: 'Client Typescript',
-          type: 'link',
-          href: '/iotajs/welcome',
-        },
-        {
-          label: 'Identity',
-          type: 'link',
-          href: '/identity.rs/introduction',
+          type: 'doc',
+          id: 'endpoints/shimmer',
+          label: 'Shimmer',
         },
         {
-          label: 'Stronghold',
-          type: 'link',
-          href: '/stronghold.rs/welcome',
+          type: 'doc',
+          id: 'endpoints/testnet',
+          label: 'Testnet',
         },
       ],
     },
+
     {
       type: 'category',
-      label: 'Node Software',
-      collapsed: true,
+      label: 'Documentation',
+      link: {
+        type: 'doc',
+        id: 'docs',
+      },
       items: [
         {
-          label: 'Introduction',
-          type: 'doc',
-          id: 'nodes/node-software',
+          type: 'link',
+          label: 'Protocol Overview',
+          href: '/introduction/welcome',
         },
         {
-          label: 'Hornet',
           type: 'link',
-          href: '/hornet/welcome',
+          label: 'Tangle Improvement Proposals (TIPS)',
+          href: '/tips',
         },
         {
           type: 'category',
-          label: 'INX Plugins',
+          label: 'Smart Contracts',
           collapsed: true,
           items: [
             {
+              label: 'Documentation',
               type: 'link',
-              label: 'Indexer',
-              href: '/inx-indexer/welcome',
-            },
-            {
-              type: 'link',
-              label: 'MQTT',
-              href: '/inx-mqtt/welcome',
+              href: '/smart-contracts/overview',
             },
             {
+              label: 'Node (WASP)',
               type: 'link',
-              label: 'PoI',
-              href: '/inx-poi/welcome',
+              href: '/smart-contracts/guide/chains_and_nodes/running-a-node',
             },
             {
               type: 'link',
-              label: 'Participation',
-              href: '/inx-participation/welcome',
+              label: 'WASP CLI',
+              href: '/smart-contracts/guide/chains_and_nodes/wasp-cli',
             },
             {
+              label: 'Schema Tool',
               type: 'link',
-              label: 'Coordinator',
-              href: '/inx-coordinator/welcome',
+              href: '/smart-contracts/guide/wasm_vm/schema',
             },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'Libraries',
+          collapsed: true,
+          items: [
             {
+              label: 'Client',
               type: 'link',
-              label: 'Faucet',
-              href: '/inx-faucet/welcome',
+              href: '/iota.rs/welcome',
             },
             {
+              label: 'Wallet.rs',
```

Comment:
Just a small thing. But is it:
```suggestion
              label: 'wallet.rs',
```
?